### DISCUSSION TITLE
Saving file template in saving options as well

### DISCUSSION THREAD

**Discussion by andrewthenew** [doublecmd/doublecmd#1982] [General]
Hi there! 
Is there any possibility to save file template (for example custom template) inside saving options? 
Because now I have to chose it every time I copy directories (with content of different types).

![dc-file-template](https://github.com/user-attachments/assets/0fd70cc4-5da3-4070-a0cb-4fb632bf140f)

**Comment by alexx2000** [comment]
Hi, I guess no.
